cmake/config/config.h.in

```diff
@@ -218,3 +218,6 @@
 /* Define to the type of an unsigned integer type wide enough to hold a
    pointer, if such a type exists, and if the system does not define it. */
 #cmakedefine uintptr_t
+
+#cmakedefine INTEL_FPGA_API_VERSION @INTEL_FPGA_API_VERSION@
```

Comment:
This doesn't need quotes too?

Reply:
You're right, it does. I'll also add numeric constants for the individual components (major, minor, revision).

Reply:
What about a struct? Can we initialize a static struct with the version numbers?